test/e2e/test_log_types.py

```diff
@@ -0,0 +1,159 @@
+import pandas as pd
+
+import layer
+from layer.clients.layer import LayerClient
+from layer.contracts.logged_data import LoggedDataType
+from layer.contracts.projects import Project
+from layer.decorators import dataset
+from test.e2e.assertion_utils import E2ETestAsserter
+
+
+def test_scalar_values_logged(
+    initialized_project: Project, asserter: E2ETestAsserter, client: LayerClient
+):
+    # when
+    layer.init(initialized_project.name)
```

Comment:
Isn't the project `initialized_project` (as the name implies) already initialised?